MapboxCoreNavigation/RouteController.swift

```diff
@@ -786,8 +797,12 @@ extension RouteController: CLLocationManagerDelegate {
         }
     }
 
-    func incrementRouteProgress() {
-        routeProgress.currentLegProgress.stepIndex += 1
+    func incrementRouteProgress(forcedStepIndex: Int? = nil) {
```

Comment:
This method is a bit of a misnomer now, since it doesn’t necessarily increment by one. A more flexible name would be `advance(to:)`, which can be called as `advance()` in the normal case.